libraries/botbuilder-core/src/transcriptLogger.ts

```diff
@@ -47,12 +53,12 @@ export class TranscriptLoggerMiddleware implements Middleware {
 
         // hook up onSend pipeline
         context.onSendActivities(
-            async (ctx: TurnContext, activities: Partial<Activity>[], next2: () => Promise<ResourceResponse[]>) => {
+            async (ctx: TurnContext, activities: Partial<Activity>[], next: () => Promise<ResourceResponse[]>) => {
```

Comment:
These random `next2` have caused me random bits of anguish, thanks for changing the name!